spec/abci/apps.md

```diff
@@ -22,6 +22,26 @@ Since Tendermint maintains four concurrent ABCI connections, it is typical
 for an application to maintain a distinct state for each, and for the states to
 be synchronized during `Commit`.
 
+### Concurrency
+
+In principle, each of the four ABCI connections operate concurrently with one
+another. This means applications need to ensure access to state is
```

Comment:
Is this tendermint state or application. Would be good to differentiate. These two separate states get conflated often